config.json

```
@@ -1393,6 +1393,17 @@
         "strings"
       ]
     },
+    {
+        "slug": "dnd-character",
+        "uuid": "58625685-b5cf-4e8a-b3aa-bff54da0689d",
+        "core": false,
+        "unlocked_by": "robot-name",
```

I made a mistake. Since `robot-name` is not a core exercise, it cannot unlock other exercises. Please revert the change I requested.
```suggestion
        "unlocked_by": null,
```

---

This is why the Travis build failed. Should be ready to merge after this.